packages/auth/src/Auth.ts

```diff
@@ -85,6 +85,10 @@ const dispatchAuthEvent = (event: string, data: any, message: string) => {
 	Hub.dispatch('auth', { event, data, message }, 'Auth', AMPLIFY_SYMBOL);
 };
 
+// as per Cognito's Documentation
+// https://tinyurl.com/c2a9f4hz
```

Comment:
I would use the official link instead
```suggestion
// https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_ListDevices.html#API_ListDevices_RequestSyntax
```

Reply:
based on the tslinting rules, the link was too long, 120 characters

Reply:
could consider disable the linting by adding a comment
```
// tslint:disable-next-line:max-line-length
```